library/src/main/java/com/getstream/sdk/chat/adapter/MediaAttachmentSelectedAdapter.java

```diff
@@ -75,6 +75,13 @@ public void bind(Attachment attachment, final OnItemClickListener listener) {
                     .solidColor(Color.BLACK)
                     .cornerRadii(cornerRadius, cornerRadius, cornerRadius, cornerRadius)
                     .build());
+
+            binding.ivMask.setShape(context, new DrawableBuilder()
```

Comment:
When uploading a file fails (ie too large) the file stays selected and clicking to unselect actually puts it back in the message (which will fail)

![image](https://user-images.githubusercontent.com/88735/66920260-3843cd80-f023-11e9-8734-c13f8a0f31f6.png)

Reply:
Fixed it.